plugins/chat/assets/javascripts/discourse/components/chat/modal/edit-channel-name.hbs

```diff
@@ -34,7 +34,7 @@
         name="channel-slug"
         class="chat-channel-edit-name-slug-modal__slug-input"
         placeholder={{this.autoGeneratedSlug}}
-        {{on "input" (action "onChangeChatChannelSlug" value="target.value")}}
+        {{on "input" this.onChangeChatChannelSlug}}
```

Comment:
So this is a little strange, the argument was never actually used in `onChangeChatChannelSlug` – it actually references `this.editedName` and doesn't do anything with `this.editedSlug`. That may or may not be correct but in any case this argument can be removed